shiny/workflowPlot/helper.R

```diff
@@ -0,0 +1,14 @@
+checkAndDownload<-function(packageNames) {
+  for(packageName in packageNames) {
+    if(!isInstalled(packageName)) {
+      install.packages(packageName,repos="http://lib.stat.cmu.edu/R/CRAN") 
+    } 
+    library(packageName,character.only=TRUE,quietly=TRUE,verbose=FALSE)
+  }
+}
+isInstalled <- function(mypkg){
+  is.element(mypkg, installed.packages()[,1])
+}
+checkAndDownload(c('plotly','scales','dplyr'))
+# devtools::install_github('hadley/ggplot2')
+# testVal = 5
```

Comment:
This should be removed. Instead, specify package dependencies in the PEcAn.visualization package DESCRIPTION file.